### ISSUE TITLE
GROUP BY does not work on field keys

### ISSUE THREAD

**Issue by redsiren204** [pgspider/influxdb_fdw#3]
On the commit "GROUP BY push down (#2)", this error occurs:
```sql
select c2/2, sum(c2) * (c2/2) from ft1 group by c2/2 order by c2/2;
ERROR:  influxdb_fdw : error parsing query: found FROM, expected identifier, string, number, bool at line 1, char 9
```
It seems that GROUP BY function can not work with all field keys on InfluxDB.
Please check and fix this issue.

**Comment by mochizk** [comment]
Thanks for the report.
Currently GROUP BY works for only tag keys.
I plan to fix this issue, but it may take a little time.

**Comment by hrkuma** [comment]
We have fixed in Release v0.3.0.
So we'd like to close this issue.
